include/nbl/video/ILogicalDevice.h

```diff
@@ -301,7 +319,9 @@ class NBL_API ILogicalDevice : public core::IReferenceCounted, public IDeviceMem
         }
 
         //! Fill out the descriptor sets with descriptors
-        virtual void updateDescriptorSets(uint32_t descriptorWriteCount, const IGPUDescriptorSet::SWriteDescriptorSet* pDescriptorWrites, uint32_t descriptorCopyCount, const IGPUDescriptorSet::SCopyDescriptorSet* pDescriptorCopies) = 0;
+        bool updateDescriptorSets(uint32_t descriptorWriteCount, const IGPUDescriptorSet::SWriteDescriptorSet* pDescriptorWrites, uint32_t descriptorCopyCount, const IGPUDescriptorSet::SCopyDescriptorSet* pDescriptorCopies);
+
+        bool freeDescriptorSets(IDescriptorPool* pool, const uint32_t descriptorSetCount, IGPUDescriptorSet *const *const descriptorSets);
```

Comment:
make it a method of the pool

Reply:
Resolved.

Reply:
they're still not methods of the descriptor pool

Reply:
`freeDescriptorSets` is. Do you want `updateDescriptorSets` to be a method of the pool as well?

Reply:
Made `updateDescriptorSets` a method of the pool also.

Reply:
> Made updateDescriptorSets a method of the pool also.

As discussed on Discord, reverting this change.

Reply:
Reverted. Resolved.

Reply:
settled in discord.